src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ObsoleteReferences.targets

```diff
@@ -0,0 +1,38 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.ObsoleteReferences.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(NETCoreSdkBundledCliToolsProps)' == ''">
+    <NETCoreSdkBundledCliToolsProps>$(MSBuildThisFileDirectory)..\..\..\Microsoft.NETCoreSdk.BundledCliTools.props</NETCoreSdkBundledCliToolsProps>
+  </PropertyGroup>
+
+  <Import Project="$(NETCoreSdkBundledCliToolsProps)" Condition="Exists('$(NETCoreSdkBundledCliToolsProps)')" />
+
+  <ItemGroup>
+    <_ReferenceToObsoleteDotNetCliTool Include="@(DotNetCliToolReference)" />
+    <DotNetCliToolReference Remove="@(BundledDotNetCliToolReference)" />
+    <_ReferenceToObsoleteDotNetCliTool Remove="@(DotNetCliToolReference)" />
+  </ItemGroup>
+
+  <Target Name="_WarnAboutObsoleteDotNetCliToolReferences"
+          BeforeTargets="CollectPackageReferences"
+          Condition=" '$(SuppressObsoleteDotNetCliToolWarning)' != 'true' ">
+    <Warning Code="DOTNET1018"
+             Text="Using DotNetCliToolReference to reference '%(_ReferenceToObsoleteDotNetCliTool.Identity)' is obsolete and can be removed from this project. This tool is bundled by default in the .NET Core SDK."
```

Comment:
This should be localized

Reply:
Happy to. How?

Reply:
We have NETSdkError task. We briefly had NETSdkWarning but I believe it was just deleted due to being unused. Resurrect it from history and use it.

Reply:
Added NETSdkWarning.